libs/authentication/website/src/lib/login-form/login-form.tsx

```diff
@@ -43,9 +44,16 @@ const loginFormSchema = z.union([
   withCredentialsFormSchema,
 ]);
 
-const autofocus = (node: HTMLElement | null) => {
+const autofocus = (node: HTMLElement | null, focusDelay?: number) => {
```

Comment:
Adding a delay is probably fixing the symptom not the cause. But fine for now (going live for OR but want to revisit that before merged into dev).